gpr.py

```diff
@@ -1,5 +1,62 @@
+import matplotlib
+matplotlib.use('agg')
+from matplotlib import cm, pyplot
+import numpy
+from sklearn import gaussian_process
+import io
+import arff
+import os
+import json
+
+
+def branin_mesh(X0, X1):
```

Comment:
I kept the routine in gpr.py and imported it in branin.py. That is because we need the dependency inside the algo file when publishing, we can't use modules (or at least I don't know of a way to import modular algorithms with c2d)